software/base/src/test/java/brooklyn/entity/AbstractEc2LiveTest.java

```diff
@@ -75,10 +75,11 @@ public void setUp() throws Exception {
         super.setUp();
     }
 
+    // Image ids for Debian: https://wiki.debian.org/Cloud/AmazonEC2Image/Squeeze
     @Test(groups = {"Live"})
     public void test_Debian_6() throws Exception {
         // release codename "squeeze"
-        runTest(ImmutableMap.of("imageId", "us-east-1/ami-0740476e", "loginUser", "admin", "hardwareId", SMALL_HARDWARE_ID));
+        runTest(ImmutableMap.of("imageId", "us-east-1/ami-5e12dc36", "loginUser", "admin", "hardwareId", SMALL_HARDWARE_ID));
```

Comment:
5.6 has problem when using curl with ssl and
It fails to download the ssh installation script.
Also it is over 3 years old and I saw that there is several implementations of AbstractEc2LiveTest which just override the test_CentOS_5_6 method with an empty one because the test is not working so I decided to make a pull request about it.